src/main/java/persistance/ClientCredentials.java

```diff
@@ -0,0 +1,62 @@
+package persistance;
+
+import javax.persistence.*;
+
+@Entity(name = "client_credentials")
+public class ClientCredentials {
```

Comment:
All of this class is missing Javadoc comments - please could you add them

Reply:
done